pkg/security/probe/probe.go

```diff
@@ -65,18 +65,24 @@ type EventHandler interface {
 	HandleEvent(event *model.Event)
 }
 
-// EventConsumerInterface represents a handler for events sent by the probe. This handler makes a copy of the event upon receipt
-type EventConsumerInterface interface {
-	ID() string
+// EventConsumerHandler represents a handler for events sent by the probe. This handler makes a copy of the event upon receipt
+type EventConsumerHandler interface {
+	IDer
 	ChanSize() int
 	HandleEvent(_ any)
 	Copy(_ *model.Event) any
 	EventTypes() []model.EventType
 }
 
+// IDer provides unique ID for each event consumer
+type IDer interface {
+	// ID returns the ID of the event consumer
+	ID() string
+}
+
```

Comment:
why did you declare twice on the same interface?

Reply:
moved all event consumer interfaces to a platform-agnostic file (without go:build tags)